src/poetry/utils/extras.py

```diff
@@ -64,3 +65,19 @@ def _extra_packages(package_names: Iterable[str]) -> Iterator[str]:
                     yield dependency_package_name
 
     return _extra_packages(extra_package_names)
+
+
+def strtobool(val: str) -> Literal[0, 1]:
```

Comment:
If we are creating our own version, I would prefer it not to be exact copy. 
- name should be `str_to_bool`, because its more readable
- if the name is "..to_bool", then it should return bool (so returns should be `True` and `False`)